code/src/components/Result.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
```

Comment:
This is impressive Peki! Its easy to follow what happens with the 3 props in the 4 calculations. Even thou my math / workout skills aren't there I understand the results part :)